composites/Plugin/Shared/components/YoastInput.js

```diff
@@ -18,7 +18,11 @@ export const YoastInputLabel = styled.label`
 `;
 
 export const YoastInputField = styled.input`
-	border: 1px solid ${ colors.$color_input_border };
-	padding: 0.75em;
-	font-size: 1em;
+	&&& {
+		box-shadow: inset 0 1px 2px rgba(0,0,0,.07);
+		border: 1px solid ${ colors.$color_input_border };
+		padding: 0.75em;
```

Comment:
I know this value is pre-existing, but it was discussed at some point because `em` values don't make much sense in this context and don't match our 8 pixels based grid size. Both in the standalone app and in Gutenberg the computed value is `9.75px` which will also be subject to browsers rounding. I'd consider to use a different value, in pixels /Cc @hedgefield